packages/discord-request/src/bucket.ts

```diff
@@ -0,0 +1,158 @@
+import type { URL } from "url";
+import type { RequestInit, Response } from "node-fetch";
+import fetch from "node-fetch";
+
+function parseBoolean(
+	bool: string | number | boolean | undefined | null
+): boolean {
+	if (bool == null) {
+		return false;
+	}
+
+	switch (bool) {
```

Comment:
Is this a basic structure for later? If not, there is no need for the empty cases (or whatever they are called), `default` will handle them.